climakitaegui/explore/warming.py

```diff
@@ -1088,3 +1088,140 @@ def fit_models_and_plots(
     plt.show()
 
     return new_params, trad_params
+
+
+class IPCCVisualize:
+    """Class for visualizing IPCC warming trajectories"""
```

Comment:
Please use `numpy` style [docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes) for classes, methods, functions, and modules.